bigtable-hbase/src/main/java/org/apache/hadoop/hbase/client/AbstractBigtableConnection.java

```diff
@@ -97,6 +97,16 @@ public void run() {
 
   private static final Set<RegionLocator> locatorCache = new CopyOnWriteArraySet<>();
 
+  private static BigtableOptions toOptions(Configuration conf) throws IOException {
+    try {
+      return BigtableOptionsFactory.fromConfiguration(conf);
+    } catch (IOException ioe) {
+      new Logger(AbstractBigtableConnection.class).error(
```

Comment:
We only need this logger if there's a startup failure in converting the configuration to the BigtableOption.  The LOG variable is a member variable so that its log statements can show exactly which type of subclass is being used.  This usage can't use the member variable, so it needs its own Logger.  Since there's no reuse, I figured there's no benefit of maintaining a reference to this logger elsewhere.